sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,219 @@
+   list in their constructor [88fab25]
+ - Fixed support for offset in `parallel_for` for host device [1958715]
+ - Added missing constructors for `sycl::buffer` class [bdfad9e]
+ - Fixed coordinate conversion for `sampler` class on host device [cd6529f]
+ - Fixed support for local accessors in debugger [fdacb75]
+ - Fixed dropping of kernel attributes when execution range rounding is used
+   [496f9a0] [677a7ea]
+ - Added support for interoperability tasks that use `get_mem()` methods with
+   Level Zero plugin [149f08d]
+ - Fixed sub-device caching in the Level Zero plugin [0b18b49]
+ - Fixed `get_native` methods to retain reference counter in case of OpenCL
+   backend [ee7e99f]
+ - Fixed sporadic failure happening due to illegal destruction of events before
+   they have been signaled [2a76b2a]
+ - Resolved a pinned host memory specific performance regression on CUDA that
+   was introduced with the host unified behavior dependent logic [3be63ab]
+ - Fixed illegal accesses that could happen when an application that uses host
+   tasks exits without waiting for host tasks completion [552a521]
+ - Fixed `sycl::event::get_info` queries that were working incorrectly when
+   called on event without an encapsulated native handle [5d5a792]
+ - Fixed compilation error with using multidimensional subscript for
+   `sycl::accessor` with atomic access mode [0bfd34e]
+ -  Fixed a crash that happened when an accessor passed to a reduction was
+    destroyed immediately after [b80f13e]
+ - Fixed `sycl::device::get_info` with `sycl::info::device::max_mem_alloc_size`
+   which was returning incorrect value in case of Level Zero backend [8dbaa53]
+
+## API/ABI breakages
+- None
+
+## Known issues
+ - GlobalWorkOffset is not supported by Level Zero backend [6f9e9a76]
+ - User-defined functions with the same name and signature (exact match of
+   arguments, return type doesn't matter) as of an OpenCL C built-in
+   function, can lead to Undefined Behavior.
+ - A DPC++ system that has FPGAs installed does not support multi-process
+   execution. Creating a context opens the device associated with the context
+   and places a lock on it for that process. No other process may use that
+   device. Some queries about the device through device.get_info<>() also
+   open up the device and lock it to that process since the runtime needs
+   to query the actual device to obtain that information.
+ - On Windows, DPC++ compiler enforces using dynamic C++ runtime for
+   application linked with SYCL library by:
+    - linking with msvcrt[d].dll when `-fsycl` switch is used;
+    - emitting an error on attempts to compile a program with static C++ RT
+      using `-fsycl` and `/MT` or `/MTd`.
+   That protects you from complicated runtime errors caused by C++ objects
+   crossing sycl[d].dll boundary and not always handled properly by different
+   versions of C++ RT used on app and sycl[d].dll sides.
```

Comment:
Should we remove it and translate that to documented -fsycl behavior on Windows?
It's not known issue it seems, it's rather how -fsycl switch is expected to work.
@v-klochkov

Reply:
I think this statement may be removed from 'known issues' list.